### PR TITLE
Introducing Swift Package Manager

### PR DESCRIPTION
**Pull request by 3ign0n**
[Swift Package Manager](https://github.com/apple/swift-package-manager) is integrated into Xcode11.
It's time to introduce SPM now.

* This branch is tested with Xcode11 beta6(11M392q).

### CODE REVIEW

**Comment by 4brunu** [comment]
@pkluz could you please take a look at this? I would like to use this lib, but I need SPM integration 🙂

File: Demo/DemoViewController.swift [hunk 1]
```diff
@@ -10,20 +10,19 @@ import UIKit
 import PKHUD
 
 class DemoViewController: UIViewController {
-    
-    let hiddenTextField = UITextField(frame: CGRect.zero)
 
+    let hiddenTextField = UITextField(frame: CGRect.zero)
```
Comment (awall1) [inline Demo/DemoViewController.swift]: Unintended unrelated changes?

Comment (3ign0n) [reply Demo/DemoViewController.swift]: Xcode 11.3.1 reports
`Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)`

スクリーンショット 2020-03-11 9 28 10

**Comment by PGLongo** [comment]
Closed via #277
